tests/unit/workflows/cassettes/test_guess_coreness_when_non_core.yaml

```diff
@@ -25,23 +25,35 @@ interactions:
       beam.", "title": "Relative Biological Effectiveness of Antiprotons the AD-4/ACE
       Experiment"}'
     headers:
-      Accept: [application/json]
-      Accept-Encoding: ['gzip, deflate']
-      Connection: [keep-alive]
-      Content-Length: ['1871']
-      Content-Type: [application/json]
+      Accept:
+      - application/json
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '1871'
+      Content-Type:
+      - application/json
     method: POST
-    uri: https://inspire-classifier.web.cern.ch/api/predict/coreness
+    uri: http://10.254.0.186:5000/api/predict/coreness
```

Comment:
how the cassettes are working since `inspire-classifier.web.cern.ch` is configured in the tests https://github.com/inspirehep/inspire-next/pull/4152/files#diff-38141e7792071032740e5e8291fbdbec4708a3172637cc035d1dbe1821cc87e3R63

Reply:
cause I set url inside tests https://github.com/inspirehep/inspire-next/pull/4152/files#diff-38141e7792071032740e5e8291fbdbec4708a3172637cc035d1dbe1821cc87e3R168